packages/web-tests/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "web-tests",
```

Comment:
Just checking, in the future if we want tests for the marketing website, will it be a separate package?

We also might want to have it as `@documenso/web-tests` 

or `@documenso/app-tests` if we can include marketing in this package in the future